sklearn/tests/test_calibration.py

```diff
@@ -179,6 +179,28 @@ def test_sample_weight(data, method, ensemble):
     assert diff > 0.1
 
 
+@pytest.mark.parametrize("method", ["sigmoid", "isotonic"])
+@pytest.mark.parametrize("ensemble", [True, False])
+def test_sample_weight_class_imbalanced(data, method, ensemble):
+    X, y = make_blobs((100, 1000), center_box=(-1, 1), random_state=42)
+
+    # Compute weigths to compensate the unbalance of the dataset
+    sample_weight = 9 * (y == 0) + 1
+
+    X_train, X_test, y_train, y_test, sw_train, sw_test = train_test_split(
+        X, y, sample_weight, random_state=42
+    )
+
+    base_estimator = LinearSVC(random_state=42)
```

Comment:
It would be good to add a preprocessor such as `StandardScaler`.

Reply:
Easy to add a ```StandardScaler``` indeed. However, no scaler is used in any of the calibration tests, even when data is generated via ```make_blobs```. Wouldn't it better not to add one for the sake of consistency?

Reply:
No, because it is a bad practice to not scale the data with SVC. If I recall why I propose it was due to a `ConvergenceWarning` that could be easily suppressed.

We should most probably do that for the other test but we can let this on the side and address it in another PR.

Reply:
done.